config.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+from logging import Logger, getLogger
+from typing import List
```

Comment:
I noticed in one of your suggestions in the earlier code review, you used an actual data type rather than one from `typing`.  (It might have been `list`.)  I thought that `typing` _should_ be used for this purpose, because it seems to have been created for this.

I'll make a note to review Python documentation about whether using actual types or those from `typing` are recommended.  If you already know of a documented recommendation, please tell me.

Reply:
Based on our watercooler conversation, etc., I'll go along with the notion that it's better to use the actual type rather than import something else from `typing`.

Reply:
Resolved in 2f0dc10.